src/js/stylesheets.js

```diff
@@ -0,0 +1,52 @@
+const BASE_THEME = "BASE_THEME"
+const THEME_VARIANT_ONE = "THEME_VARIANT_ONE"
+
+const THEME_CLASSNAMES = {
+  [BASE_THEME]:        "",
+  [THEME_VARIANT_ONE]: "theme-one"
+}
+
+const THEME_OPTIONS = [
+  { value: BASE_THEME, label: "Base Theme" },
+  { value: THEME_VARIANT_ONE, label: "Theme #1" }
+]
+
+const STORAGE_KEY = "ocw-theme"
+
+const getTheme = () => window.localStorage.getItem(STORAGE_KEY) || BASE_THEME
+
+const setTheme = theme => window.localStorage.setItem(STORAGE_KEY, theme)
+
+const setThemeClass = theme => {
+  $("html").removeClass()
```

Comment:
This is causing Travis to fail.  I realize `$` is jquery, so to make eslint happy we should probably ignore `eslint no-undef` for these lines or redefine it within this file by assigning it to a local variable.